docs/guide/user-guide.md

```diff
@@ -1 +1,109 @@
 # User guide
+
+In the user guide you can find a description of all components belonging to 
+DeepESDL.
+
+There are three components of DeepESDL: 
+1. [DeepESDL JupyterLab](#deepesdl-jupyterlab)
+2. [DeepESDL xcube viewer](#deepesdl-xcube-viewer)
+3. [DeepESDL datacube resources](#deepesdl-datacube-resources)
+
+
+# DeepESDL JupyterLab
+
+## Basic usage
+
+This section provides a brief introduction for users to the basic features of
+the JupyterLab environment as deployed on the DeepESDL system. 
+For more in-depth documentation on the various components, see the links in the
+[‘Further information’ section](#further-information).
+
+### Logging in and starting the JupyterLab profile
+
+To use the DeepESDL JupyterLab environment, navigate to
+<https://deep.earthsystemdatalab.net/> with a web browser (a recent version of
+Firefox, Chrome, or Safari is recommended).
+
+DeepESDL uses a GitHub to authenticate, so if you are already registered as a 
```

Comment:
DeepESDL uses GitHub to authenticate
User must be registered first. We thus need a sentence lie: Before first usage, we will have to register you with the system. To this end, we kindly ask you to duly compete the registration dialogue (link)/write as an email/...